Assignments/stu1/assignments.py

```diff
@@ -103,5 +103,9 @@ def ex7():
 # Place your functions here...
 #
 def sort_people(people_list, x, y):
-    people_list.sort(key=lambda y: y[x])
+    people_list.sort(key=lambda z: z[x], reverse=True)
+    if x == sorted(y,reverse=False):
+        return 'Ascending'
```

Comment:
We don't need to return `Ascending` or `Descending`.

Here are a few use cases that the `sort_people()` function has to support:

`sort_people(people_list, 'weight', 'desc')`  ==> Returns a list sorted in desc order by weight
`sort_people(people_list, 'weight', 'asc')`  ==> Returns a list sorted in asc order by weight
`sort_people(people_list, 'age', 'asc')`  ==> Returns a list sorted in asc order by age
`sort_people(people_list, 'age', 'desc')`  ==> Returns a list sorted in desc order by age
`sort_people(people_list, 'name', 'desc')`  ==> Returns a list sorted in desc order by name

and so on...